aggregator/src/env.ts

```diff
@@ -11,11 +12,26 @@ import nil from "./helpers/nil.ts";
 export const RPC_URL = requireEnv("RPC_URL");
 
 export const ORIGIN = requireEnv("ORIGIN");
-export const PORT = requireIntEnv("PORT");
+
+export const PORT = (() => {
+  const portOverride = Deno.env.get("PORT_OVERRIDE");
+
+  if (portOverride !== nil) {
+    const port = Number(portOverride);
+    assert(Number.isFinite(port));
+    return port;
+  }
+
+  return requireIntEnv("PORT");
+})();
 
 export const USE_TEST_NET = requireBoolEnv("USE_TEST_NET");
 
-export const NETWORK_CONFIG_PATH = requireEnv("NETWORK_CONFIG_PATH");
+export const NETWORK_CONFIG_PATH = (
+  Deno.env.get("NETWORK_CONFIG_PATH_OVERRIDE") ??
+    requireEnv("NETWORK_CONFIG_PATH")
+);
```

Comment:
Is there a reason we need these overrides? The container should have access to the env vars from the env file, so an agg operator can just update those as need be.

Reply:
I can't say we 'need' them, but the reason they're there is to allow you to run the 'same' config in docker. Without these overrides, you need to make a dedicated env file when running docker.